README.md

```diff
@@ -239,7 +271,9 @@ IE 11
 not dead
 ```
 
-We recommend using a **minifier** like [csso](https://github.com/css/csso) to compress your final compiled CSS and **sourcemaps** like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
+**Minfiers and sourcemaps:** We recommend using a minifier like [csso](https://github.com/css/csso) to compress your final compiled CSS and sourcemaps like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
+
+Using [`uswds-compile`](https://github.com/uswds/uswds-compile) includes all these requirements by default and can be a good way to build a reliable compile workflow into your project.
```

Comment:
On first read, it was a little confusing to me if the requirements referenced are for minifier/sourcemaps or all Sass compilation. I also wonder if it would be helpful to move this note about `uswds-compile` to the top of the Sass requirements section.

One idea:
```suggestion
Using [`uswds-compile`](https://github.com/uswds/uswds-compile) includes all these Sass compilation requirements by default and can be a good way to build a reliable compile workflow into your project.
```